packages/terra-layout/src/LayoutSlidePanel.module.scss

```
@@ -39,45 +39,47 @@
   }
 
   .panel.is-tiny {
-    width: var(--layout-slide-panel-phone-panel-width, calc(100% - 3.571rem));
+    width: var(--terra-layout-slide-panel-phone-panel-width, calc(100% - 3.571rem));
```

Let's replace `phone` with `tiny`
```suggestion
    width: var(--terra-layout-slide-panel-tiny-panel-width, calc(100% - 3.571rem));
```

@tbiethman @StephenEsser Is `tiny` the right name for when this style is applied according to the breakpoint name update we just did?

---

Fixed in https://github.com/cerner/terra-framework/pull/429/commits/82c3fd5b63ab11a574da1f8323fde4731425153b